src/App.tsx

```diff
@@ -1,27 +1,76 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos, getUsers } from './api/api';
+
+const defaultUser = {
+  id: 0,
+  createdAt: '',
+  updatedAt: '',
+  name: '',
+  username: '',
+  email: '',
+  phone: '',
+  website: '',
+};
```

Comment:
you don't need this fake data. Just set null in state and draw loader if no user yet